sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -75,6 +79,27 @@
     private PermissionManager permissionManager;
     */
 
+	CompletionListener subManagerListener = new CompletionListener() {
+		boolean allSucceeded = true;
+		int subManagerCount = 1; // Update per amount of sub managers implemented
```

Comment:
This seems like an error waiting to happen. We should find a less brute force way.